eng/update-dependencies.cmd

```diff
@@ -0,0 +1,4 @@
+@echo off
+powershell -ExecutionPolicy ByPass -NoProfile %~dp0build.ps1 -inittools
+powershell -ExecutionPolicy ByPass -NoProfile %~dp0common\msbuild.ps1 %~dp0..\build.proj %*
```

Comment:
the script is called update-dependencies, but it looks to me like it doesn't specify that target when building build.proj.

Reply:
yeah I was debating about that but I decided to let Maestro continue to pass all the arguments see https://github.com/dotnet/versions/pull/380/files. I really just needed a wrapper project that would initialize all the tools and then run any custom target on build.proj. We could either give this a more general name or make it more specific but for now it will do the job.

Reply:
I got confused by the same thing, when reading the script I was like where is the target that updates the dependencies being passed when building. Maybe call it "init-tools-run-target.cmd" ? I don't know but by the name it can definitely get confusing.

Reply:
The script will go away complete once we finish all our work because we will switch to the new darc dependency update mechanism.

Reply:
Ok, then don't bother updating it, I thought it was staying longer, 😄